### DISCUSSION TITLE
How do I develop the chongluadao-extension?

### DISCUSSION THREAD

**Discussion by nghiahung-dev** [7zones/chongluadao-extension#73] [Q&A]
Hi, How can I develop the chongluadao-extension? I want to improve the library. I tried to develop the library but don't know where it started. Can you explain the problem?

**Comment by dshongphuc** [comment]
Hi @nnhjs , I'm really appreciate that you're trying to help.
Currently, we don't have a clear plan or task to improve this extension, But it is still many things that need help. So first of all, please try to clone the project and import it to your browser as "Load Unpackged". And then try to play around with it to have an overview of how it works.
After that, I suggest you to find some bugs with it. Could be layout issues, error when sending API, wrong calculation with the score,...

If you found any, please open an issue and explain it, then please fix it and create a PR. You can tag me to the "reviewer" field of that PR so I can review it. 

Thanks again my friend :)

![Screenshot from 2022-07-29 14-24-45](https://user-images.githubusercontent.com/19908411/181706682-4ad9c612-71b5-4612-881a-3e26f77f2327.png)

**Reply by nghiahung-dev** [reply]
> Thanks for the answer. I will try the way above soon and create PR when have issue :D. I wish you can help you if I have problem. ❤️
